### PR TITLE
Adds regular and security biosuit crates.

### PR DESCRIPTION
**Pull request by GoodG1448**
Not sure if the price is cheap or not for something that can let you walk in a plasma flood as long as you have internals and someone hasn't set the room on fire. Both regular and security biosuits are the same which means they don't have bullet/fire/laser protection.

:cl:

- You can now order regular and security biosuits from cargo.

### CODE REVIEW

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -1018,6 +1018,19 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
...
 	group = "Security"
 
+/datum/supply_packs/secbiosuits
+	name = "Biohazard Emergency Biosuits"
+	contains = list(/obj/item/clothing/head/bio_hood/security,
+					/obj/item/clothing/head/bio_hood/security,
+					/obj/item/clothing/head/bio_hood/security,
+					/obj/item/clothing/suit/bio_suit/security,
+					/obj/item/clothing/suit/bio_suit/security,
+					/obj/item/clothing/suit/bio_suit/security,
```
Comment (boy2mantwicethefam) [inline code/datums/supplypacks.dm]: add a ) at the end, ditto for the normal biosuits to fix conflict

**Comment by jknpj** [comment]
might be a tad too cheap

**Comment by GoodG1448** [comment]
85$ seems fine. They aren't radiation suits that let you walk into rad storms to justify going over 100$.

**Comment by FudgePucker** [comment]
If you do a weak ass baby plasma flood with room temp plasma it deserves a counter.
Install igniters under tables or some shit, git gud.

**Pull request merged**